src/main/resources/SnakemakeBundle.properties

```diff
@@ -4,8 +4,12 @@ INSP.GROUP.snakemake=Snakemake
 # SnakemakeResourcesUnnamedArgsInspection
 INSP.NAME.resources.unnamed.args=Only keyword arguments allowed in 'resources' section (e.g. 'threads=4').
 
+# SnakemakeRuleRedeclarationInspection
+INSP.NAME.rule.redeclaration=This rule name is already used by another rule.
+
 # SnakemakePositionalArgsAfterKeywordArgsInspection
 INSP.NAME.positional.args.after.keyword.args=Syntax error: positional argument follows keyword argument.
 
-# SnakemakeRuleRedeclarationInspection
-INSP.NAME.rule.redeclaration=This rule name is already used by another rule.
+# SnakemakeRuleKeywordsAfterExecutionInspection
+INSP.NAME.rule.keywords.after.execution=No rule keywords allowed after execution keywords.
+INSP.NAME.rule.keywords.after.$0=No rule keywords allowed after ''{0}'' keyword.
```

Comment:
Please avoid using special symbols for bundle keys. Better is to use keys with clear readable names, e.g. here is just an inspection message and the previous one is inspection title.